specs/capella/p2p-interface.md

```diff
@@ -59,8 +59,10 @@ See Capella [state transition document](./beacon-chain.md#beaconblockbody) for f
 
 This topic is used to propagate signed bls to execution change messages to be included in future blocks.
 
+Let `system_clock_epoch` be an epoch number according to the local system clock of a node.
 The following validations MUST pass before forwarding the `signed_bls_to_execution_change` on the network:
 
+- _[IGNORE]_ `system_clock_epoch >= CAPELLA_FORK_EPOCH`.
```

Comment:
The older p2p specs use `current_epoch`. Good to be consistent? 

One example:
```
Clients MUST keep a record of signed blocks seen on the epoch range
`[max(GENESIS_EPOCH, current_epoch - MIN_EPOCHS_FOR_BLOCK_REQUESTS), current_epoch]`
where `current_epoch` is defined by the current wall-clock time,
```